core/src/main/java/com/sequenceiq/cloudbreak/controller/StackCreatorService.java

```diff
@@ -245,7 +245,7 @@ public StackV4Response createStack(User user, Workspace workspace, StackV4Reques
                         "Select the correct image took {} ms");
                 stackRuntimeVersionValidator.validate(stackRequest, imgFromCatalog.getImage(), stackType);
                 Stack newStack = measure(() -> stackService.create(
-                            stack, platformString, imgFromCatalog, user, workspace, Optional.ofNullable(stackRequest.getResourceCrn())),
```

Comment:
why we removed platformstring?

Reply:
because it's not used, just a leftover. I like to honor `boy scout rule`